internal/rbd/controllerserver.go

```diff
@@ -1559,3 +1559,35 @@ func (cs *ControllerServer) ControllerExpandVolume(
 		NodeExpansionRequired: nodeExpansion,
 	}, nil
 }
+
+func (cs *ControllerServer) ControllerPublishVolume(
+	ctx context.Context,
+	req *csi.ControllerPublishVolumeRequest,
+) (*csi.ControllerPublishVolumeResponse, error) {
+	// A dummy publish implementation to mimic a successful attach operation being a NOOP
```

Comment:
a description of the function should be above the function, that counts for `ControllerUnpublishVolume` too.